core/templates/dev/head/domain/question/EditableQuestionBackendApiService.js

```diff
@@ -19,12 +19,47 @@
 oppia.constant(
   'EDITABLE_QUESTION_DATA_URL_TEMPLATE',
   '/question_editor_handler/data/<question_id>');
+oppia.constant(
+  'QUESTION_CREATION_URL', '/question_editor_handler/create_new');
+oppia.constant(
+  'QUESTION_SKILL_LINK_URL_TEMPLATE',
+  '/manage_question_skill_link/<question_id>/<skill_id>');
 
 oppia.factory('EditableQuestionBackendApiService', [
   '$http', '$q', 'EDITABLE_QUESTION_DATA_URL_TEMPLATE',
-  'UrlInterpolationService',
-  function($http, $q, EDITABLE_QUESTION_DATA_URL_TEMPLATE,
-      UrlInterpolationService) {
+  'UrlInterpolationService', 'QUESTION_CREATION_URL',
+  'QUESTION_SKILL_LINK_URL_TEMPLATE',
+  function(
+      $http, $q, EDITABLE_QUESTION_DATA_URL_TEMPLATE,
+      UrlInterpolationService, QUESTION_CREATION_URL,
+      QUESTION_SKILL_LINK_URL_TEMPLATE) {
+    var _createQuestion = function(
+        skillId, questionDict, successCallback, errorCallback) {
+      var postData = {
+        question_dict: questionDict
+      };
+      $http.post(QUESTION_CREATION_URL, postData).then(function(response) {
```

Comment:
Oh, also, for this, I don't think we should do two POST requests. We should make a single POST request to create the question. The backend should update multiple models as needed.

Reply:
Done.